packit_service/celerizer.py

```diff
@@ -20,10 +20,12 @@ def celery_app(self):
             password = getenv("REDIS_PASSWORD", "")
             port = getenv("REDIS_SERVICE_PORT", "6379")
             db = getenv("REDIS_SERVICE_DB", "0")
+            celery_beckend = getenv("REDIS_CELERY_BECKEND", "1")
```

Comment:
what is this value exactly? :thinking: I was confused seeing the default is `1` after reading the name

```suggestion
            celery_backend = getenv("REDIS_CELERY_BECKEND", "1")
```

Reply:
This is a new redis db, to be used as celery backend. The 0 db is  already taken by "Packit", so I am using the next available.